Include/internal/pycore_frame.h

```diff
@@ -75,7 +75,8 @@ typedef struct _PyInterpreterFrame {
     _PyStackRef *stackpointer;
     uint16_t return_offset;  /* Only relevant during a function call */
     char owner;
-    char visited;
+    char visited:4;
+    char is_entry_frame:4;
```

Comment:
This is unnecessary. `frame->owner == FRAME_OWNED_BY_CSTACK` is true iff the frame is an entry frame.

Hypothetically would could add frames for builtin functions for better debugging and tracebacks, but we can easily enough add a new `enum _frameowner` if necessary.
Maybe replace `FRAME_OWNED_BY_CSTACK` with `FRAME_OWNED_BY_INTERPRETER_ENTRY`?